app/assets/stylesheets/pages/_contact.scss

```diff
@@ -1,4 +1,77 @@
+// .collapsing for custom transition with bootstrap
+.collapsing {
+  -webkit-transition-duration: 3.35s;
+  transition-duration: 3.35s;
+  min-height: 120px;
+}
+
 .biographie {
   margin : 40px;
   margin-left: 0px;
+
+   p {
+    margin: 0px;
+   }
+}
+
+.contact-wrapper {
+  display: flex;
+  justify-content: space-between;
+}
+
+// EXHIBITIONS BOXES
+
+.exhibitions-box {
+ width: 500px;
+ box-shadow: $box-shadow;
+ height: fit-content;
+
+  h2 {
+    text-align: center;
+    font-weight: bold;
+  }
+
+  p {
+   padding-bottom: 0px 4px;
+   margin: 0px 8px;
+  }
+}
+
+.exhibitions-showed {
+
+  &.swap {
+    display: none;
+  }
+}
+
+.exhibitions-showed,
+.exhibitions-hidden {
+  background: white;
+}
+
+.exhibitions-hidden {
+  max-height: 500px;
+  overflow: scroll;
+}
+
+.show-more {
+  cursor: clicker;
+  text-align: center;
+
+  p {
+    margin: 0px;
+  }
+}
+
+// EXHIBITIONS TEXT
+
+.exhibition-year {
+  margin: 0 4px;
+  font-weight: bolder;
+}
+
+.exhibition-title,
+.exhibition-place {
+  font-style: oblique;
```

Comment:
Yes it is a good idea to distinguish the different type of infos for lisibility

Reply:
it give us capability to change each padding/margin :)